lib/index.js

```diff
@@ -355,9 +356,25 @@ class Mcdev {
      * @param {boolean} [fromRetrieve] optionally deploy whats defined via selectedTypesArr + keyArr directly from retrieve folder instead of from deploy folder
      * @returns {Promise.<Object.<string,TYPE.MultiMetadataTypeMap>>} deployed metadata per BU (first key: bu name, second key: metadata type)
      */
-    static async deploy(businessUnit, selectedTypesArr, keyArr, fromRetrieve = false) {
+    static async deploy(
+        businessUnit,
+        selectedTypesArr,
+        keyArr,
+        fromRetrieve = false
+        // ,
```

Comment:
I tried testing --refresh option on asset deploy, it does not seem to work, shouldn't we pass this it as a parameter in deploy (and all related deploy functions eg deployBU )?

Alternatively read Util.OPTIONS.refresh in postDeploy since we set refresh and execute as we do with --changeKeyField?
![image](https://github.com/Accenture/sfmc-devtools/assets/79999614/b49f1bf4-d62e-4130-b4c5-28138d6c3e40)

Reply:
Refresh and fromRetrieve should both be read from Util.OPTIONS instead of via function parameter. 
If --refresh did not work we might have a regression here.

Reply:
> Refresh and fromRetrieve should both be read from Util.OPTIONS instead of via function parameter. If --refresh did not work we might have a regression here.

Ok, tested again and submitted an issue
https://github.com/Accenture/sfmc-devtools/issues/988